### ISSUE TITLE
Remove automatically added hero banner field on home/landing

### ISSUE THREAD

**Issue by lburack** [NCIOCPL/cgov-digital-platform#1625]
This is a ticket to remove the automatically added hero banner field on the home/landing page. This should be distinct entity that users choose to add. This is just like how we handle paragraphs like Related Resources, Citations, etc. 

The way it is implemented now, it's easy to miss the Remove button (since the user did not add the entity themselves) and it then appears to be a required field (see @BrendanBeltzNIH comments/screenshot below).

**Comment by amydugan** [comment]
@lburack and @zhuomingao - When Nano is migrated, will it be impacted by having banner required on home/landing content type? Or will the content move as is through the scripts? In other words, does it absolutely have to be done before migrating content? Or could it wait until after launch?

**Comment by zhuomingao** [comment]
As for migration, I think home landing can be migrated without a banner image.

**Comment by lburack** [comment]
@amydugan @zhuomingao – In my testing last week, I found that a banner image and tagline are required on the home/landing page content type. I created #1625 to make the banner optional.

**Comment by BrendanBeltzNIH** [comment]
I think there's a bit of confusion on this one. It's not a required field. You can see the remove button to the right. It's the subfields on the paragraph that are required. I.e., if you have a hero banner it needs to have a desktop and tablet image, and a tagline. 


Perhaps the real change you're looking for is to not have the field prepopulated with a paragraph by default:

![image](https://user-images.githubusercontent.com/34315527/57301941-1c1d2700-70a8-11e9-8717-b99c4a95e183.png)

**Comment by lburack** [comment]
Thanks, @BrendanBeltzNIH. I totally missed that Remove button. Since the entity was automatically added (and I did not add it myself), and then I saw the asterisk, I assumed it was a required field. I've updated the description of the ticket to reflect the new request.

@amydugan @VictoriaSunNIH  - I don't think this needs to be High Priority anymore. I thought this was a blocker since Nano doesn't have a hero banner. But the hero banner isn't required. It's just a bad user interface. This could probably even be fixed after launch, since I don't think we'll be creating new home/landing pages in the immediate future.
